src/data/issue_extraction.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Can you run these cells as well so that the output is visible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>